space-concordia-robotics/robotics-prototype
# Light 563

# Assignee Section

## Description
- Adds code so the wheel teensy can control the activity light.
- Improve the organization of the debug mode for serial

### Steps for Testing
1. Upload the wheel teensy code, ensure the status indicator light is plugged in
2. Switch to this branch on the on-board computer, and ensure the main repo is in this branch
3. Restart ROS if a branch switch was needed
4. SSH into the rover
5. Issue the following command: ```rostopic pub /rover_command std_msgs/String "data: 'blink_toggle 1'" ```
6. The status indicator light should be flashing red

closes #563

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
